packages/cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@subql/cli",
   "description": "cli for subquery",
-  "version": "2.1.1-1",
+  "version": "3.0.0",
```

Comment:
Just confirm we want to have a major bump? due to TS strict is a breaking change?